gulpfile.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+
+sass.compiler = require('node-sass');
+
+gulp.task('sass', function () {
+    return gulp.src('assets/scss/*.scss')
```

Comment:
I think it will produce many css files here. Maybe we don't need them, since we import `style.css` only. We could cut it down to `assets/scss/style.scss`

Reply:
> I think it will produce many css files here. Maybe we don't need them, since we import `style.css` only. We could cut it down to `assets/scss/style.scss`

Not really, if they use scss partials (I mentioned about it in review), it will create just one css file while compiling. Usually in gulp task: styles (here sass), you also put autoprefixer, plumber, cssnano, etc. and then create gulp.watch to watch this task, so I think it's necessary to see changes in all files and in which file problem occurred, otherwise you will see you have a problem in style.scss. But it depends on a configuration, right now it's a simple one.

Reply:
We do use scss partials and the the code is compiling into one css file. I also added in gulp-task to create the css sourcemap to easier find the problems.